static_files/css/closure/closure.css

```diff
@@ -26,6 +26,9 @@
 @import 'menuitem.css';
 @import 'splitpane.css';
 
+@import '../../external/closure/roundedtab.css';
```

Comment:
Seems file roundedtab.css is missing in the PR.

Reply:
I think the latest commit is the right way to do it. They're just vanilla closure css files, but it looks like we have our own customized copies of other closure css stuff. Let me know if that's not the proper way to copy them from third_party or if we're already pulling in some minified version of closure that I missed somewhere.